src/hotspot/os/linux/os_linux.cpp

```diff
@@ -1728,7 +1728,7 @@ void * os::dll_load(const char *filename, char *ebuf, int ebuflen) {
   static  Elf32_Half running_arch_code=EM_SH;
 #elif  (defined RISCV)
   static  Elf32_Half running_arch_code=EM_RISCV;
-#elif  (defined LOONGARCH)
+#elif  (defined LOONGARCH64)
```

Comment:
Maybe we should also rename the `LOONGARCH` macro here at this line?https://github.com/openjdk/jdk/blob/master/src/hotspot/os/linux/os_linux.cpp#L1735